README.md

```diff
@@ -51,6 +51,40 @@ This behavior could also be used for going into the move less mode again and mak
 
 After such an action the folding will be immediately removed. It is only intended to be used for navigation without moving.
 
+## Customizing the navigation shortcuts
```

Comment:
Do you like this or the one in doc/move-less.txt?

Reply:
both are good, but i like this one inside the readme.md more ;)
but it seems like it's still described that you could partially overwrite the key set which should not longer be the case if I understood you correctly.